packages/modules/imodel-browser/src/components/noResults/NoResults.tsx

```diff
@@ -4,21 +4,45 @@
  *--------------------------------------------------------------------------------------------*/
 import "./NoResults.scss";
 
-import { SvgImodelHollow } from "@itwin/itwinui-icons-react";
+import { SvgImodelHollow, SvgSearch } from "@itwin/itwinui-icons-react";
 import { Text } from "@itwin/itwinui-react";
 import React from "react";
 
 export interface NoResultsProps {
   /** Displayed text */
   text: string;
+  subtext?: string;
+  isSearchResult?: boolean;
 }
 
 /** Pre-formatted empty result page */
-export const NoResults = ({ text }: NoResultsProps) => {
-  return (
-    <Text variant="leading" isMuted={true} className={"iac-no-results"}>
-      <SvgImodelHollow />
-      <span>{text}</span>
-    </Text>
-  );
+export const NoResults = ({
+  text,
+  subtext,
+  isSearchResult = false,
+}: NoResultsProps) => {
+  const renderNoresultsComponent = () => {
+    return (
+      <Text variant="leading" isMuted={true} className={"iac-no-results"}>
+        <SvgImodelHollow />
+        <span>{text}</span>
+      </Text>
+    );
+  };
+
+  const renderNoSearchResultsComponent = () => {
+    return (
+      <div className="iac-no-search-results-container">
+        <div className="iac-no-search-results">
+          <SvgSearch />
+          <Text variant="leading">{text}</Text>
+          {subtext && <Text>{subtext}</Text>}
+        </div>
+      </div>
+    );
+  };
+
+  return isSearchResult
+    ? renderNoSearchResultsComponent()
+    : renderNoresultsComponent();
```

Comment:
These are 2 independent component, if the new organisation of NoSearchComponent is validated by UX, we should apply it to NoResults component, which is what I met by improving NoResults instead of creating a separate component.

NoResults tried to override placements of iTwinUI component and "work" them to look a bit like we wanted. If the proper look is acheived by having the Svg outside of  the text and the location by enclosing in2 divs, please do the same thing so the 2 structures are the same.

(Something like below, but validate with UX if the text should stay muted or not.)

```suggestion
  return (
      <div className="iac-no-search-results-container">
        <div className="iac-no-search-results">
          {isSearchResult ? <SvgSearch /> : <SvgImodelHollow> }
          <Text variant="leading">{text}</Text>
          {subtext && <Text>{subtext}</Text>}
        </div>
      </div>
    );
```

Reply:
I talked with our UX and was given a go ahead for this change (from muted to leading text for no result text), as we will be now consistent with both the pattern meaning in text and style, the only difference being the icon and the text message its good to go forward, I have pushed the changes accordingly